src/main/java/org/broadinstitute/hellbender/tools/exome/NormalizeSomaticReadCounts.java

```diff
@@ -20,18 +21,39 @@
 import java.util.List;
 
 /**
- * Normalizes read counts given the PanelOfNormals.
+ * Normalizes read counts given the PanelOfNormals (PoN).
  *
- * <p> Dev note:  If this is extended to use spark, please be wary that the parallelization in tangent normalization is
+ * <p> A note to developers:  If this is extended to use Spark, please be wary that the parallelization in tangent normalization is
  * by case sample, which may not yield benefits for most use cases (which are one sample)  </p>
  *
  * @author Valentin Ruano-Rubio &lt;valentin@broadinstitute.org&gt;
+ *
+ * <h3>Examples</h3>
+ * <p>
+ *     The command encompasses empirically determined parameters for TCGA project data.
```

Comment:
This does not make a lot of sense here.  Especially, since the pon file in the example does not actually exist.  And it's not really accurate anyway.